### PR TITLE
[system] Use the only unique Dockerfile for nightly build

### PR DESCRIPTION
**Pull request by hallelujah**
We are moving the Dockerfile.on_prem to Dockerfile

Requires: https://github.com/3scale/porta/pull/2132

### CODE REVIEW

**Review by guicassolato — APPROVED** [review]

**Review by eguzki — APPROVED** [review]

**Review by miguelsorianod — APPROVED** [review]

**Comment by miguelsorianod** [comment]
Thanks @hallelujah :+1:
Let's wait until the required PR is merged in Porta and the nightly built to merge this

**Pull request merged**
